scripts/prepare.js

```diff
@@ -2,18 +2,14 @@
 const path = require('path');
 const shell = require('shelljs');
 const chalk = require('chalk');
-const fs = require('fs');
+const fs = require('fs-extra');
 const log = require('npmlog');
+const readPkgUp = require('read-pkg-up');
 const { babelify } = require('./utils/compile-babel');
 const { tscfy } = require('./utils/compile-tsc');
 
-function getPackageJson(modulePath) {
-  // eslint-disable-next-line global-require,import/no-dynamic-require
-  return require(path.join(modulePath, 'package.json'));
-}
-
-function removeDist() {
-  shell.rm('-rf', 'dist');
+async function removeDist() {
+  await fs.remove('dist');
```

Comment:
making things async-y so that hopefully we're blocking the main thread less, and we can do more with less resources.